tests/testthat/test-share_spec.R

```diff
@@ -4,19 +4,20 @@ data("raman_hdpe")
 tmp <- file.path(tempdir(), "OpenSpecy-testthat")
 dir.create(tmp, showWarnings = F)
 
+wo_meta <- raman_hdpe
+wo_meta$metadata$spectrum_identity <- NULL
+
 test_that("share_text() gives expected output", {
-  expect_message(share_spec(raman_hdpe,
-                            metadata = c(user_name = "Win Cowger",
-                                         spectrum_type = "FTIR",
-                                         spectrum_identity = "PE",
-                                         license = "CC BY-NC"),
-                            share = tmp
-  ))
-  expect_warning(
-    expect_message(share_spec(raman_hdpe, share = tmp))
-  )
-  expect_error(share_spec(raman_hdpe, metadata = c("a", "b", "c"),
-                          share = tmp))
+  share_spec(raman_hdpe, share = tmp) |>
+    expect_message()
+  file.path(tmp, raman_hdpe$metadata$session_id,
+            paste0(raman_hdpe$metadata$file_id, ".yml")) |>
+    file.exists() |>
+    expect_true()
```

Comment:
If we want to test that the upload to AWS works, @wincowgerDEV, you could add the S3 credentials `"AWS_ACCESS_KEY_ID"` and `"AWS_SECRET_ACCESS_KEY"` to the secrets in the OpenSpecy-package repo. I can then retrieve them for testing without exposing them.

Reply:
Good call, I can get that added next week!